src/HealthChecks.Publisher.ApplicationInsights/ApplicationInsightsPublisher.cs

```diff
@@ -23,8 +23,9 @@ class ApplicationInsightsPublisher
         private static readonly object sync_root = new object();
         private readonly bool _saveDetailedReport;
         private readonly bool _excludeHealthyReports;
+        private readonly bool _trackExceptions;
 
-        public ApplicationInsightsPublisher(string instrumentationKey = default, bool saveDetailedReport = false, bool excludeHealthyReports = false)
+        public ApplicationInsightsPublisher(string instrumentationKey = default, bool saveDetailedReport = false, bool excludeHealthyReports = false, bool trackExceptions = true)

```

Comment:
Where is _tracekExpcetions assigned?

Reply:
This has been fixed in 4d93ae545bcaac05e3b7c4f2fd6e0ed686993f73.